.circleci/build_autoinst.sh

```diff
@@ -18,12 +18,12 @@
 set -ex
 
 destdir=${1:-../os-autoinst}
-sha=${2}
+sha=${2:-./autoinst.sha}
 
 echo Building os-autoinst $destdir $sha
 git clone https://github.com/os-autoinst/os-autoinst.git "$destdir"
 ( cd "$destdir"
-[ -z "$sha" ] || git checkout $sha
+git checkout $sha
```

Comment:
Looks like `$sha` is possibly a file path. So I'm not sure whether this will work. I'd also quote the parameter.